### ISSUE TITLE
therarbg.py is returning bytes when it should be returning MB

### ISSUE THREAD

**Issue by Sophist-UK** [BurningMop/qBittorrent-Search-Plugins#25]
Size is scraped from HTML here: https://github.com/BurningMop/qBittorrent-Search-Plugins/blob/main/therarbg.py#L130-L132 .

I see this was fixed in https://github.com/BurningMop/qBittorrent-Search-Plugins/commit/831d95e123bebc9669ac508ba567644cdf24e5f5 however updates are not being downloaded by qbittorrent which is not seeing the new version.

**Comment by BurningMop** [comment]
I probably forgot to update the version, let me try that

**Comment by BurningMop** [comment]
I just checked, and the version was updated to 1.3. AFAIR, there is no automatic update of the plugins, you'll need to manually install version 1.3

**Comment by Sophist-UK** [comment]
qbt does have a check for update, but it didn't find the new versions - I suspect that this is only for official plugins. So I deleted and reinstalled from this repo.

But the [qbt wiki page for unofficial plugins](https://github.com/qbittorrent/search-plugins/wiki/Unofficial-search-plugins) still had the old version numbers and dates, so I updated these. Whether this helps with the updater finding the new versions I have no idea.
